local_planner/src/nodes/local_planner_node.h

```diff
@@ -94,7 +94,7 @@ class LocalPlannerNode {
   const ros::Duration pointcloud_timeout_hover_ = ros::Duration(0.4);
   const ros::Duration pointcloud_timeout_land_ = ros::Duration(10);
 
-  ros::Time last_wp_time_;
+  ros::Time last_pc_time_;
```

Comment:
`wp` stood for waypoint. What does `pc` stand for?

Reply:
pointcloud. Should I expand?

Reply:
Unless it gets really long and difficult to work with, I prefer not to have abbreviations :+1: